addons/mail/static/src/models/autocomplete_input_view.js

```diff
@@ -11,6 +11,17 @@ registerModel({
         'discussViewOwnerAsMobileAddItemHeader',
         'messagingMenuOwnerAsMobileNewMessageInput',
     ]],
+    recordMethods: {
+        /**
+         * @param {FocusEvent} ev
+         */
+        onFocusin(ev) {
+            if (this.chatWindowOwnerAsNewMessage) {
+                this.chatWindowOwnerAsNewMessage.onFocusInNewMessageFormInput(ev);
```

Comment:
I wonder if we actually want to keep the method, or move its code here?
But otherwise diff for the handler in this PR is ok.

Reply:
Same reply as previous review comment.

Reply:
Not sure why duplicate comment